src/content-helper/editor-sidebar/related-top-posts/provider.ts

```diff
@@ -183,6 +183,7 @@ export class RelatedTopPostsProvider {
 
 		if ( PostFilterType.Tag === filterType ) {
 			// Get post's first tag.
+			// @ts-ignore
```

Comment:
Ignoring this isn't an ideal solution, but the intention is to refactor/move the code soon. At that point, we'll see if we can find a better alternative.